SPECS/k3s/vendor_build.patch

```diff
@@ -100,167 +100,6 @@ index fb538350d0..3712ef2abf 100755
  popd
 -cp -vf ./build/src/github.com/opencontainers/runc/runc ./bin/
 +cp -vf ./$RUNC_DIR/runc/runc ./bin/
-diff --git a/scripts/build.orig b/scripts/build.orig
```

Comment:
Just to satisfy my curiosity- why are we dropping this part of the patch?

Reply:
This line doesn't suppose to be there, we do the patch for the existing patches for the older version. I tried using the older patch again in local I don't see this line coming up. Not sure why it was there in the version which I am replacing but it is not required.